lib/teleterm/api/proto/v1/cluster.proto

```diff
@@ -0,0 +1,87 @@
+// Copyright 2021 Gravitational, Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package teleport.terminal.v1;
+
+option go_package = "github.com/gravitational/teleport/lib/teleterm/v1";
+
+// Cluster describes cluster fields
+message Cluster {
+    // uri is the cluster resource URI
+    string uri = 1;
+    // name is the cluster name
+    string name = 2;
+    // connected indicates if connection to the cluster has been established
```

Comment:
The value of this field is basically the result of `Cluster.Connected()`. I believe this is comment slightly inaccurate, I'll change it so that it says what the comment for `Cluster.Connected()` says.

https://github.com/gravitational/teleport/blob/c681dd5896e388883ac3afbde394800f2d6ad91e/lib/teleterm/clusters/cluster.go#L51-L54

https://github.com/gravitational/teleport/blob/c681dd5896e388883ac3afbde394800f2d6ad91e/lib/teleterm/apiserver/handler/handler_clusters.go#L87-L92